README.md

```diff
@@ -37,6 +37,7 @@ Organizations bringing diversity to tech.
 - [Open Sourcing Mental Illness](https://osmihelp.org/) - Open Sourcing Mental Illness is a non-profit corporation dedicated to raising awareness, educating, and providing resources to support mental wellness in the tech and open source communities.
 - [ParadigmIQ](https://www.paradigmiq.com/) - Paradigm offers a range of workshops, training and assessments aiding in diversity, inclusion, belonging, retention and more.
 - [People Of Creativity](http://www.peopleofcreativity.xyz/) - Organization that focuses on creating a community that shares opportunities, experiences and ideas ultimately bettering the representation of minorities throughout creative industries.
+- [PHP Women](http://phpwomen.org - An inclusive & global network providing support within the PHP community.
```

Comment:
I think we have a typo here with the markdown link we must fix before merging, right? (missing `)` at the end of the link) 😄 

```
[PHP Women](http://phpwomen.org)
```

Reply:
oops! thanks for catching that :)